cmd/auth/main.go

```diff
@@ -1 +1,56 @@
-package auth
+package main
+
+import (
+	"context"
+	"log"
+	"monorepa/pkg/auth"
+	"os"
+	"os/signal"
+)
+
+func init() {
+	// serever
+	os.Setenv("SERVER_HOST", "")
```

Comment:
You should not do setenv here. Env vars are used to tune the app externally, and here you are overwriting these variables. In the future, I hope to see all env variables set in the makefile.